CHANGES.md

```diff
@@ -10,6 +10,15 @@ updated. Only the first match gets replaced, so it's fine to leave the old
 ones in. -->
 --------------------------------------------------------------------------------
 
+## 2.0.2 (<span actions:bind='release-date'>Upcoming</span>)
+
+### Fixes
+
+[#2608](https://github.com/metomi/rose/pull/2608) - Fix problems with the
```

Comment:
Should this be here?

Reply:
Yes, I forgot to add a changelog entrey in that PR